### ISSUE TITLE
Adapt to new bajes conventions

### ISSUE THREAD

**Issue by GCArullo** [GCArullo/JenpyROQ#14]
[Flagged by @RoxGamba]

Future `bajes` releases change the waveform interface, giving the following error:

```

Traceback (most recent call last):
  File "/cluster/miniconda3-py38/lib/python3.8/runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/cluster/miniconda3-py38/lib/python3.8/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/beegfs/re89yow/fd_inj_test/fd-inj/lib/python3.8/site-packages/JenpyROQ-0.0.1-py3.8.egg/JenpyROQ/__main__.py", line 146, in <module>
  File "/beegfs/re89yow/fd_inj_test/fd-inj/lib/python3.8/site-packages/JenpyROQ-0.0.1-py3.8.egg/JenpyROQ/jenpyroq.py", line 89, in __init__
  File "/beegfs/re89yow/fd_inj_test/fd-inj/lib/python3.8/site-packages/JenpyROQ-0.0.1-py3.8.egg/JenpyROQ/waveform_wrappers.py", line 390, in __init__
TypeError: __init__() takes 1 positional argument but 4 were given

```

This only happens for the `dev` branch at the moment. Once this branch is merged into master, JenpyROQ interface should also be updated.
